editoast/src/views/paced_train.rs

```diff
@@ -88,6 +111,147 @@ async fn delete(
     Ok(axum::http::StatusCode::NO_CONTENT)
 }
 
+#[derive(Debug, IntoParams, Deserialize)]
+struct PacedTrainIdParam {
+    id: i64,
+}
+
+/// Get a paced train by its ID
+#[utoipa::path(
+    get, path = "",
+    tag = "timetable, paced_train",
+    params(PacedTrainIdParam),
+    responses(
+        (status = 204, body = PacedTrainResult, description = "The requested paced train")
+    )
+)]
+async fn get_by_id(
+    State(_db_pool): State<DbConnectionPoolV2>,
+    Extension(_auth): AuthenticationExt,
+    Path(PacedTrainIdParam {
+        id: _paced_train_id,
+    }): Path<PacedTrainIdParam>,
+) {
+}
```

Comment:
```suggestion
) -> Result<Json<PacedTrainResult>> {
    todo!()
}
```

We could add the result type.